packages/components/Menu/src/MenuCallout/MenuCallout.types.ts

```diff
@@ -0,0 +1,22 @@
+import type { ICalloutProps, ICalloutTokens } from '@fluentui-react-native/callout';
+
+export const menuCalloutName = 'MenuCallout';
+
+// Support for anchorRect and beakWidth will come at a later time.
+// Omitting dismissBehaviors as it doesn't seem to make sense as a token
+export type MenuCalloutTokens =
+  | Omit<ICalloutTokens, 'anchorRect' | 'beakWidth' | 'dismissBehaviors'> & {
+      /**
+       * The props for the corner radius for the Modal MenuPopover
+       * @platform android
+       */
+      cornerRadius?: number;
+
```

Comment:
Just to confirm this, are these tokens or props?

Reply:
tokens.

Reply:
These are tokens.